src/cpp/session/modules/SessionHelp.R

```diff
@@ -795,38 +800,14 @@ options(help_type = "html")
       file <- helpfiles[[1]]
       path <- dirname(file)
       dirpath <- dirname(path)
-      pkgname <- basename(dirpath)
+      package <- basename(dirpath)
       
-      query <- paste("/library/", pkgname, "/html/", basename(file), ".html", sep = "")
+      query <- sprintf("/library/%s/html/%s.html", package, basename(file))
       html <- suppressWarnings(tools:::httpd(query, NULL, NULL))$payload
    }
    
-   # resolve associated package from helpfile path -- note that help file
```

Comment:
This should be dead code -- we now set `package` in the two branches above, and (conceptually) a help topic should always be associated with a package anyhow.